docs/configuration/features/text_features.md

```diff
@@ -47,197 +48,52 @@ preprocessing:
-
-The [spaCy](https://spacy.io) based tokenizers are functions that use the powerful tokenization and NLP preprocessing models provided the library.
-Several languages are available: English (code `en`), Italian (code `it`), Spanish (code `es`), German (code `de`), French (code `fr`), Portuguese (code `pt`), Dutch (code `nl`), Greek (code `el`), Chinese (code `zh`), Danish (code `da`), Dutch (code `el`), Japanese (code `ja`), Lithuanian (code `lt`), Norwegian (code `nb`), Polish (code `pl`), Romanian (code `ro`) and Multi (code `xx`, useful in case you have a dataset containing different languages).
-For each language different functions are available:
-
-- `tokenize`: uses spaCy tokenizer,
-- `tokenize_filter`: uses spaCy tokenizer and filters out punctuation, numbers, stopwords and words shorter than 3 characters,
-- `tokenize_remove_stopwords`: uses spaCy tokenizer and filters out stopwords,
-- `lemmatize`: uses spaCy lemmatizer,
-- `lemmatize_filter`: uses spaCy lemmatizer and filters out punctuation, numbers, stopwords and words shorter than 3 characters,
-- `lemmatize_remove_stopwords`: uses spaCy lemmatize and filters out stopwords.
-
-In order to use these options, you have to download the the spaCy model:
-
-```
-python -m spacy download <language_code>
-```
-
-and provide `<language>_<function>` as `tokenizer` like: `english_tokenizer`, `italian_lemmatize_filter`, `multi_tokenize_filter` and so on.
-More details on the models can be found in the [spaCy documentation](https://spacy.io/models).
+!!! note
+    If a text feature's encoder specifies a huggingface model, then the tokenizer for that model will be used
+    automatically.
 
 ## Text Input Features and Encoders
 
 Text input feature parameters are
 
-- `encoder` (default `parallel_cnn`): encoder to use for the input text feature. The available encoders come from [Sequence Features](#sequence-input-features-and-encoders) and these text specific encoders: `bert`, `gpt`, `gpt2`, `xlnet`, `xlm`, `roberta`, `distilbert`, `ctrl`, `camembert`, `albert`, `t5`, `xlmroberta`, `flaubert`, `electra`, `longformer` and `auto-transformer`.
-- `level` (default `word`): `word` specifies using text words, `char` use individual characters.
+- `encoder` (default `parallel_cnn`): encoder to use for the input text feature. The available encoders include encoders
+used for [Sequence Features](#sequence-input-features-and-encoders) as well as encoders from the huggingface
+transformers library: `bert`, `gpt`, `gpt2`, `xlnet`, `xlm`, `roberta`, `distilbert`, `ctrl`, `camembert`, `albert`,
+`t5`, `xlmroberta`, `flaubert`, `electra`, `longformer` and `auto-transformer`.
+- `level` (default `word`): `word` specifies using text words/symbols, `char` use individual characters.
 - `tied_weights` (default `null`): name of the input feature to tie the weights of the encoder with. It needs to be the name of a feature of the same type and with the same encoder parameters.
```

Comment:
I will go back and review this in the document I edited where I didn't update tied_weights -> tied
```suggestion
- `tied` (default `null`): name of the input feature to tie the weights of the encoder with. It needs to be the name of a feature of the same type and with the same encoder parameters.
```